src/kits/tracker/ContainerWindow.cpp

```diff
@@ -545,6 +508,7 @@ BContainerWindow::BContainerWindow(LockingList<BWindow>* list,
 	fIsTrash(false),
 	fInTrash(false),
 	fIsPrinters(false),
+	fIsDeskWindow(isDeskWindow),
```

Comment:
I know the class is named that way, but I would prefer either fIsDesktop, or even fIsDesktopWindow if it has to be.